projects/core/components/primitive-textfield/primitive-textfield-options.ts

```diff
@@ -0,0 +1,27 @@
+import {InjectionToken} from '@angular/core';
+import {TuiHorizontalDirection} from '@taiga-ui/core/types';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface PrimitiveTextfieldOptions {
+    readonly iconAlign: TuiHorizontalDirection;
+    readonly iconCleaner: PolymorpheusContent;
+    readonly postfix: string;
+}
+
+// TODO: remove in ivy compilation
+export const PRIMITIVE_TEXTFIELD_ICON_CLEANER = 'tuiIconCloseLarge';
+
+/** Default values for primitive textfield options */
+export const TUI_PRIMITIVE_TEXTFIELD_DEFAULT_OPTIONS: PrimitiveTextfieldOptions = {
+    iconAlign: 'right',
+    iconCleaner: PRIMITIVE_TEXTFIELD_ICON_CLEANER,
+    postfix: '',
```

Comment:
Not sure `postfix` makes sense in DI config. It is something that most likely changes from control to control and being able to set something else as default, other than an empty string seems excessive to me. Do you have a use case?

Reply:
Not really and I guess you're right about that. Will be fixed up today